codebundles/rds-mysql-conn-count/README.md

```diff
@@ -0,0 +1,95 @@
+# CodeBundle - RDS MySQL Connection Count
+
```

Comment:
We can tell that these configs are placeholder YAML. one needs to modify them according to need and then paste them to the platform side.